### ISSUE TITLE
Cannot tab into menu when using dropdown-append-to-body attribute

### ISSUE THREAD

**Issue by jfarago** [angular-ui/bootstrap#4388]
[Here](http://plnkr.co/edit/1vJ1tE?p=preview) is a plunker to reproduce.

To reproduce, click on the menu and hit tab. I would expect the first item in the menu to have fucus but the next button does. If you remove the attribute it works as expected.

**Comment by wesleycho** [comment]
This will be annoying to fix - we basically need to hijack the tab key being pressed and put the focus on the first element.

**Comment by icfantv** [comment]
Closing in favor of #4772.
